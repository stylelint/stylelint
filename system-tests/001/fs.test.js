'use strict';

const stylelint = require('../../lib');
const { caseConfigFile, caseFiles, prepForSnapshot } = require('../systemTestUtils');

const CASE_NUMBER = '001';

it('fs - valid sanitize.css and their config', async () => {
	expect(
		prepForSnapshot(
			await stylelint.lint({
				files: caseFiles(CASE_NUMBER),
				configFile: caseConfigFile(CASE_NUMBER),
			}),
		),
	).toMatchSnapshot();
}, 10000);
