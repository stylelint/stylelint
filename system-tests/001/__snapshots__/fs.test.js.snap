// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fs - valid sanitize.css and their config 1`] = `
Object {
  "errored": false,
  "output": Array [
    Object {
      "deprecations": Array [],
      "errored": false,
      "invalidOptionWarnings": Array [],
      "parseErrors": Array [],
      "warnings": Array [],
    },
  ],
  "results": Array [
    Object {
      "deprecations": Array [],
      "errored": false,
      "ignored": undefined,
      "invalidOptionWarnings": Array [],
      "parseErrors": Array [],
      "warnings": Array [],
    },
  ],
}
`;
